pkg/cloud/cloud.go

```diff
@@ -166,6 +166,9 @@ var (
 
 	// VolumeNotBeingModified is returned if volume being described is not being modified
 	VolumeNotBeingModified = fmt.Errorf("volume is not being modified")
+
+	// ErrInternal is an error that is returned when "impossible" events occur.
+	ErrInternal = errors.New("Internal error (this should be impossible!)")
```

Comment:
As this error is used today, I agree that it should be impossible. Honestly, I wouldn't have bothered to write that check in the fist place, since it's bordering on `if 1 + 1 != 2` territory.

That said, error messages should be specific or actionable. "`(this should be impossible!)`" is neither. Suggest deleting the parenthetical.

Reply:
Changed to `(submit a bug report)`, which is actionable